mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -52,7 +56,9 @@ CLASS net/minecraft/unmapped/C_tahhkgty net/minecraft/entity/vehicle/AbstractMin
 		ARG 1 offset
 	METHOD m_xtdpuoft willHitBlockAt (Lnet/minecraft/unmapped/C_hynzadkk;)Z
 		ARG 1 pos
+	METHOD m_xzvoogyl getContainedBlockOrEmpty ()Ljava/util/Optional;
```

Comment:
noticing an inconsistency here. not an issue for this PR, but i'm not sure whether we should prefer `OrEmpty` or `Optional` as a suffix for optional values. I personally prefer optional, but I think I would ascribe that to experience and a newer programmer would appreciate OrEmpty. I'll open an issue!

Reply:
This is related to #665
I could just broaden that to all getters, unless you'd prefer a separate issue.

I prefer `OrEmpty` because it reads more naturally. `getContainedBlockOptional` isn't natural at all; `getContainedOptionalBlock` is better, but it's not as simple as a suffix, and imo it still isn't as natural as `OrEmpty`.